### DISCUSSION TITLE
Making a figure list

### DISCUSSION THREAD

**Discussion by AlexanderLambrecht** [typst/typst#552] [Q&A]
I would like to make a figure list, showing numbering, captions and pagenumbers, just like the outline. How could this be done?

**Comment by laurmaedje** [comment]
This is now possible, see the updated documentation of the outline function.

**Reply by AlexanderLambrecht** [reply]
Thank you very much for implementing this, I'm currently writing my thesis using typst, it works great, this was the only point I was struggling with, and it is a minor point at that. Good luck on the continued development of typst!!!

**Reply by laurmaedje** [reply]
Thanks to @Dherse for implementing this!

**Reply by dscso** [reply]
https://typst.app/docs/reference/meta/outline/
thanks a lot for your work guys!
